core/src/navigation/GlobalSearchCentered.html

```diff
@@ -0,0 +1,499 @@
+        >
+          {#if clearSearchFieldBtn}
+          <button
+            aria-label="button-decline"
+            class="fd-shellbar__button fd-button"
+            on:click="{clearSearchField}"
+          >
+            <i class="sap-icon--decline"></i>
+          </button>
+          {/if}
+          <button aria-label="button-search" class="fd-shellbar__button fd-button">
+            <i class="sap-icon--search"></i>
+          </button>
+        </span>
+        {/if} {/if}
+      </div>
+      {#if !isCustomSearchRenderer}
+      <div
+        class="fd-popover__body fd-popover__body--right luigi-search-popover__body"
+        aria-hidden="{!displaySearchResult}"
+      >
+        <nav class="fd-menu">
+          {#if searchResult}
+          <ul
+            class="fd-menu__list fd-menu__list--top"
+            bind:this="{luigiCustomSearchItemRenderer__slotContainer}"
+          >
+            {#each searchResult as result, index}
+            <li
+              class="fd-menu__item luigi-search-result-item__{index}"
+              on:click="{(event) => onSearchResultItemSelected(result, event)}"
+              on:keyup="{(event) => handleKeydown(result, event)}"
+              tabindex="0"
+            >
+              {#if !isCustomSearchResultItemRenderer}
+              <a class="fd-menu__link" on:click|preventDefault="{() => {}}">
+                <div class="fd-product-switch__text">
+                  <div class="fd-product-switch__title">{result.label}</div>
+                  <div class="fd-product-switch__subtitle">
+                    {result.description}
+                  </div>
+                </div>
+              </a>
+              {:else} {@html
+              renderCustomSearchItem(result,luigiCustomSearchItemRenderer__slotContainer,
+              index)} {/if}
+            </li>
+            {/each}
+          </ul>
+          {/if}
+        </nav>
+      </div>
+      {:else}
+      <div bind:this="{luigiCustomSearchRenderer__slot}"></div>
+      {/if}
+    </div>
+  </div>
+</div>
+{#if !isSearchFieldVisible}
+<div class="lui-global-search-btn">
+  <div on:click|stopPropagation="{() => {}}">
+    <button
+      class="fd-button fd-shellbar__button"
+      aria-haspopup="true"
+      aria-expanded="{!isSearchFieldVisible}"
+      on:click="{toggleSearch}"
+      data-testid="luigi-search-btn-desktop"
+    >
+      <i class="sap-icon sap-icon--search"></i>
+    </button>
+  </div>
+</div>
+{/if} {#if isSearchFieldVisible}
+<div
+  class="lui-global-search-cancel-btn {isSearchFieldVisible?'lui-global-search-cancel-btn--active':''}"
+>
+  <div on:click|stopPropagation="{() => {}}">
+    <button
+      class="fd-button fd-shellbar__button"
+      aria-haspopup="true"
+      aria-expanded="{!isSearchFieldVisible}"
+      on:click="{toggleSearch}"
+      data-testid="luigi-search-cancel-btn"
+    >
+      {$getTranslation(cancelBtn)}
+    </button>
+  </div>
```

Comment:
<button
      class="fd-button fd-shellbar__button"
      aria-haspopup="true"
      aria-expanded="{!isSearchFieldVisible}"
      on:click|stopPropagation="{()=> toggleSearch()}"
```

Do you think this would work as well?